charts/k8s-watcher/values.yaml

```diff
@@ -48,6 +48,18 @@ helm:
   enableActions: true
   volumeSizeLimit: "256Mi"
 
+supervisor:
+  enabled: false
+  image:
+    repository: public.ecr.aws/g8r5w0b5/supervisor
+    pullPolicy: IfNotPresent
+  resources:
+    requests:
+      cpu: 0.1
+      memory: 256Mi
```

Comment:
Do we need that much? This means we need 512MB just to start our agent

Reply:
Probably need less. We will only need memory when we actually fetch data to send the crash report then release it back

Reply:
I'll tweak everything before enabling by default